lib/init.coffee

```diff
@@ -8,12 +8,19 @@ module.exports =
       type: 'string'
       title: 'Path to the javac executable'
       default: 'javac'
+    classpath:
+      type: 'string'
```

Comment:
Could this possibly be an array? See linter-clang: https://github.com/AtomLinter/linter-clang/blob/master/lib/main.js#L114-L116